apps/studio/data/config/jwt-secret-updating-status-query.ts

```diff
@@ -29,7 +29,8 @@ export async function getJwtSecretUpdatingStatus(
 
   if (error) handleError(error)
 
-  const meta = data.jwtSecretUpdateStatus
+  // @ts-expect-error API typing issue here
+  const { meta } = data.jwtSecretUpdateStatus
```

Comment:
```suggestion
  const meta = data.jwtSecretUpdateStatus?.meta ?? data.jwtSecretUpdateStatus
```

This way it'll work without any updates when API is fixed.